lib/reek/smells/uncommunicative_parameter_name.rb

```diff
@@ -60,10 +56,9 @@ def examine_context(ctx)
 
       private
 
-      def bad_name?(ctx, name)
-        sanitized_name = sanitize name
-        return false if sanitized_name == '*' || accept_names(ctx).include?(sanitized_name)
-        reject_patterns(ctx).any? { |pattern| sanitized_name.match pattern }
+      def acceptable_name?(context: raise, name: raise)
```

Comment:
This looks identical to UncommunicativeMethodName. Perhaps we can merge these after these two refactorings?